### PR TITLE
Refactor Group Name to fence-agents-remediation

### PR DESCRIPTION
**Pull request by razo7**
- Switching from `fence-agents` to `fence-agents-remediation` files
- Create new bundle directory with updated Medik8s fields (e.g., `provider`)
- Apply  `fence-agents` to `fence-agents-remediation` group changes in medik8s.io domain

### CODE REVIEW

**Review by clobrano — COMMENTED** [review]
Confirm
/lgtm
Giving others a chance to review as well, feel free to unhold
/hold

:)

File: config/manifests/bases/fence-agents-remediation.clusterserviceversion.yaml [hunk 1]
```diff
@@ -14,18 +14,17 @@ spec:
...
-      name: fenceagentsremediationtemplates.fence-agents.medik8s.io
+      name: fenceagentsremediationtemplates.fence-agents-remediation.medik8s.io
       version: v1alpha1
-  description: A mechanism designed around an existing set of upstream fencing agents
-    for environments with a traditional API end-point (eg. IPMI) for power cycling
-    cluster nodes
-  displayName: Fence Agents Remediation Operator
+  description: Run existing set of upstream fencing agents for environments with a
+    traditional API end-point (eg. IPMI) for power cycling cluster nodes.
+  displayName: fence-agents-remediation
```
Comment (slintes) [inline config/manifests/bases/fence-agents-remediation.clusterserviceversion.yaml:27]: I think display name can stay the old nicer value, not?

File: config/manifests/bases/fence-agents-remediation.clusterserviceversion.yaml [hunk 2]
```diff
@@ -43,21 +42,24 @@ spec:
   - supported: true
     type: AllNamespaces
   keywords:
+  - medik8s
   - fencing
-  - remediation
   - auto-healing
   - recovery
   - high-availability
+  - fecne-agents
```
Comment (slintes) [inline config/manifests/bases/fence-agents-remediation.clusterserviceversion.yaml:50]: typo

File: config/manifests/bases/fence-agents-remediation.clusterserviceversion.yaml [hunk 3]
```diff
@@ -43,21 +42,24 @@ spec:
...
+  - fence-agents-remediation
+  - remediation
+  - far
   - baremetal
-  - FAR
   links:
-  - name: Fence Agents
-    url: https://fence-agents.domain
+  - name: Fence Agents Remediation
+    url: https://fence-agents-remediation.domain
```
Comment (slintes) [inline config/manifests/bases/fence-agents-remediation.clusterserviceversion.yaml:57]: Can we have a real URL please?

File: config/crd/bases/fence-agents.medik8s.io_fenceagentsremediations.yaml [hunk 4]
```diff
@@ -5,9 +5,9 @@ metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.8.0
   creationTimestamp: null
-  name: fenceagentsremediations.fence-agents.medik8s.io
+  name: fenceagentsremediations.fence-agents-remediation.medik8s.io
```
Comment (slintes) [inline config/crd/bases/fence-agents.medik8s.io_fenceagentsremediations.yaml:8]: I think this file should be deleted, there is a new one

File: config/crd/bases/fence-agents.medik8s.io_fenceagentsremediationtemplates.yaml [hunk 5]
```diff
@@ -5,9 +5,9 @@ metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.8.0
   creationTimestamp: null
-  name: fenceagentsremediationtemplates.fence-agents.medik8s.io
+  name: fenceagentsremediationtemplates.fence-agents-remediation.medik8s.io
```
Comment (slintes) [inline config/crd/bases/fence-agents.medik8s.io_fenceagentsremediationtemplates.yaml:8]: I think this file should be deleted, there is a new one

File: bundle/manifests/fence-agents-controller-manager-metrics-service_v1_service.yaml [hunk 6]
```diff
@@ -1,17 +0,0 @@
-apiVersion: v1
```
Comment (slintes) [inline bundle/manifests/fence-agents-controller-manager-metrics-service_v1_service.yaml:1]: I think we want to keep this service?

File: bundle/manifests/fence-agents-metrics-reader_rbac.authorization.k8s.io_v1_clusterrole.yaml [hunk 7]
```diff
@@ -1,10 +0,0 @@
-apiVersion: rbac.authorization.k8s.io/v1
```
Comment (slintes) [inline bundle/manifests/fence-agents-metrics-reader_rbac.authorization.k8s.io_v1_clusterrole.yaml:1]: I think we want to keep this role?

**Review by slintes — CHANGES_REQUESTED** [review]

File: bundle/manifests/fence-agents-controller-manager-metrics-service_v1_service.yaml (hunk 6 above)
Comment (razo7) [reply bundle/manifests/fence-agents-controller-manager-metrics-service_v1_service.yaml:1]: When I [changed the group](https://github.com/medik8s/fence-agents-remediation/pull/7/files#diff-d1fb402f1269ba7396dca9cbd03b44ac055874669bce1d0f239fa2f54ea3584bR15) for the CRDs, CSV and etc. I also deleted the old bundle directory so any old files would be deleted.
IIRC the bundle directory is created and updated after we run [`make bundle`](https://github.com/medik8s/fence-agents-remediation/blob/main/Makefile#L261).
After I deleted the bundle directory and created a new CRDs, CSV, and etc. then this service and `fence-agents-metrics-reader_rbac.authorization.k8s.io_v1_clusterrole.yaml` wouldn't be created. Therefore, IIUC these files are redundant or I am missing something

File: config/crd/bases/fence-agents.medik8s.io_fenceagentsremediationtemplates.yaml (hunk 5 above)
Comment (razo7) [reply config/crd/bases/fence-agents.medik8s.io_fenceagentsremediationtemplates.yaml:8]: Correct :)

File: config/crd/bases/fence-agents.medik8s.io_fenceagentsremediations.yaml (hunk 4 above)
Comment (razo7) [reply config/crd/bases/fence-agents.medik8s.io_fenceagentsremediations.yaml:8]: Correct :)

File: config/manifests/bases/fence-agents-remediation.clusterserviceversion.yaml (hunk 3 above)
Comment (razo7) [reply config/manifests/bases/fence-agents-remediation.clusterserviceversion.yaml]: Done

File: bundle/manifests/fence-agents-metrics-reader_rbac.authorization.k8s.io_v1_clusterrole.yaml (hunk 7 above)
Comment (razo7) [reply bundle/manifests/fence-agents-metrics-reader_rbac.authorization.k8s.io_v1_clusterrole.yaml:1]: See my above comment

**Comment by clobrano** [comment]
/lgtm
Giving others a chance to review as well, feel free to unhold
/hold

**Comment by clobrano** [comment]
/lgtm
Giving others a chance to review as well, feel free to unhold
/hold

File: api/v1alpha1/groupversion_info.go [hunk 8]
```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 // Package v1alpha1 contains API Schema definitions for the fence-agents v1alpha1 API group
```
Comment (mshitrit) [inline api/v1alpha1/groupversion_info.go:17]: Nit: update comment

File: bundle/manifests/fence-agents-controller-manager-metrics-service_v1_service.yaml (hunk 6 above)
Comment (slintes) [reply bundle/manifests/fence-agents-controller-manager-metrics-service_v1_service.yaml:1]: Oh, my bad, didn't see that they were redundant before, for whatever reason. 👍🏼

**Review by slintes — APPROVED** [review]

**Pull request merged**